README.md

```diff
@@ -160,6 +161,20 @@ Don't see the service you are looking for? We are constantly adding support for
 - **Identity** - The necessary tokens, refresh tokens, or other secrets that are used to authorize API calls to a _service_ on behalf of a _user_.
 - **Tag** - A key-value pair, for example ("userId", "user-123"). A number of tags can be associated with an _identity_. EveryAuth enables you to lookup an _identity_ or _identities_ associated with a specific set of _tags_.
 
+## Security
+
+EveryAuth is backed by [Fusebit](https://fusebit.io).  Fusebit is a SOC2-compliant integration Provider
+as-a-Service ("iPaaS") that provides a broad and full-featured integration platform.  All data stored in
+Fusebit is:
+  * Encrypted at rest using AES-256 encryption
+  * Encrypted in transit using TLS 1.2
+  * Secured by a powerful authorization engine that powers the Fusebit integration platform itself, trusted
+  and used already in production by Fusebit customers
+  * Identity backed by either private keys or OAuth 2
+
+Fusebit has a strong committment to maintain a high standard for security practices, and welcomes any bug
+reports or security advisories.  Please contact support@fusebit.io for any further questions.
```

Comment:
Fusebit is committed to maintaining a high standard of security practices and welcomes any bug reports or security advisories. Don't hesitate to get in touch with support@fusebit.io for any further questions.

Reply:
Done.